index.html

```diff
@@ -11872,9 +11872,14 @@ <h2>Excluding Elements from the Accessibility Tree</h2>
 	  <h2>Including Elements in the Accessibility Tree</h2>
           <p>If not already excluded from the accessibility tree per the rules above in <a href="#tree_exclusion">Excluding Elements in the Accessibility Tree</a>, user agents MUST provide an <a>accessible object</a> in the <a>accessibility tree</a> for <abbr title="Document Object Model">DOM</abbr> <a>elements</a> that meet any of the following criteria:</p>
           <ul>
-            <li>Text nodes</li>
-            <li>Elements that may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> <a>event</a>.</li>
-            <li>Elements that are focusable, or have an ID <a>attribute</a> and an ancestor with the <pref>aria-activedescendant</pref> attribute that matches the implicit or explicit <a>semantics</a> of the required context role. In either case, the element may receive focus and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> focus event.</li>
+            <li>Text nodes.</li>
```

Comment:
I realize that the above is ancient text which was merely moved from Core-AAM as-is, but....

As long as we're clarifying things, what exactly is a text node? Given `<p>foo</p>` is the "text node" the `p` element itself, or the static text leaf node with "foo"? On my platform (ATK), we want the `p` accessible but NOT the static text leaf node. Mind you, some platforms (AXAPI) *do* expose the static text leaf node.

Long way of saying that if we're going to clean this text up, we might as well either clarify or remove "text node" from the user agents MUST provide an accessible object for....

Reply:
One possible resolution, if we don't want to explicitly define "text nodes" is to change the MUST to a MAY for text nodes and then add a MUST for the other items.

Reply:
The ARIA spec has this for [text node](http://w3c.github.io/aria/#dfn-text-node):
> Type of DOM node that represents the textual content of an attribute or an element. A Text node has no child nodes.

So that's the "leaf". So I moved "Text nodes" out of the list, and added a MAY paragraph afterwards.